apis/core/lib/handlers/column-mapping.ts

```diff
@@ -13,3 +15,24 @@ export const post = protectedResource(shaclValidate, asyncMiddleware(async (req,
   res.header('Location', columnMapping.value)
   await res.dataset(columnMapping.dataset)
 }))
+
+export const put = protectedResource(shaclValidate, asyncMiddleware(async (req, res) => {
+  const columnMapping = await updateColumnMapping({
+    resource: await req.resource(),
+  })
+
+  await res.dataset(columnMapping.dataset)
+}))
+
+export const remove = labyrinth.protectedResource(
+  asyncMiddleware(async (req, res) => {
+    /* TODO
+    const columnMapping = req.hydra.resource.term
+
+    await deleteColumnMapping({
+      resource: columnMapping,
+    })
+
+    res.sendStatus(204) */
```

Comment:
Let's return anything for now. Otherwise the requests will never end until time out
```suggestion
    res.sendStatus(204) */
    
    res.sendStatus(500)
```